libs/@blockprotocol/type-system/crate/Makefile.toml

```diff
@@ -174,8 +174,11 @@ private = true
 install_crate = { crate_name = "cargo-nextest", version = "0.9.28", binary = "cargo", test_arg = ["nextest", "--version"] }
 
 [tasks.install-wasm-pack]
-private = true
-install_crate = { crate_name = "wasm-pack", version = "0.10.3", binary = "wasm-pack", test_arg = ["--version"] }
+script = """
```

Comment:
Is there a reason you made this not `private`?

Also I appreciate this install approach is recommended but unfortunately a very large amount of the WASM-Pack docs are actually severely out of date. There _were_ large problems with installing wasm-pack like this on MacOS (maybe only M1), I _think_ this is the associated issue https://github.com/rustwasm/wasm-pack/issues/952 but there actually seems to be some progress made on it.

I'm not sure I have capacity to test it right now, so I think we have two options
- Make sure that installing through this method works fine on M1, and that the README recommends you install it this way (ideally just through a command using `cargo make`
- Make sure that this is limited to CI, jobs you can run from the README don't accidentally call this, and make sure this job is very difficult to accidentally call

Reply:
I tested this locally in a few variations and everything work as expected.

Removing the `private = true` was an oversight, but I also don't see a reason to not have it public

Reply:
Do we know how users should update wasm-pack after installing it like this?

Reply:
They can update it by just running this command

Reply:
Does this command _check_ for updates, or does it just reinstall every time?

Wondering if `cargo make build` is going to install every time for example (as this is a dependency to the task)

Reply:
My comment was badly phrased, sorry. This will only install if `wasm-pack` was not installed, this is what the check does). The user would need to remove the `wasm-pack` before. Writing this reveals that this is not optimal, I can change this to check for a specific version and install it if the version does not match (That's mainly the behavior of `cargo-make`).

Reply:
If that's not too hard to do, I'd appreciate that

Reply:
Should be doable fairly quick 🚀

Reply:
Solved by using `wasm-pack` as dev-dependency